Childrens-Social-Care-CPD-Tests/Contentful/Renderers/ListRendererTests.cs

```diff
@@ -90,13 +91,59 @@ public void List_Renders()
         actual.Should().Be("<ul class=\"HtmlEncode[[govuk-list govuk-list--bullet]]\"><li>AAA</li><li>BBB</li></ul>");
     }
 
+    [Test]
+    public void Ordered_List_Renders()
+    {
+        // arrange
+        var stringWriter = new StringWriter();
+        var list = new List()
+        {
+            NodeType = "ordered-list",
+            Content = new List<IContent>
+            {
+                new ListItem
+                {
+                    Content = new List<IContent>
+                    {
+                        new Paragraph
+                        {
+                            Content = new List<IContent> { new Text() }
+                        }
+                    }
+                },
+                new ListItem
+                {
+                    Content = new List<IContent>
+                    {
+                        new Paragraph
+                        {
+                            Content = new List<IContent> { new Hyperlink() }
+                        }
+                    }
+
+                }
+            }
+        };
+        _textLinkRenderer.Render(Arg.Any<Text>()).Returns(new HtmlString("AAA"));
```

Comment:
It's a test which replicates the buggy behaviour for which the branch was created to resolve, so it does the job it was intended to do.  I'm perhaps missing something, because I don't understand your point about paragraph filtering.  Maybe a call?

Reply:
For record: we had a call, agreed that the tests in this module as a whole aren't wrong, but aren't 100% right, and that we have higher priorities at the moment than rewriting.